meta-sokol-flex-distro/conf/distro/sokol-flex.conf

```diff
@@ -382,9 +382,6 @@ FLEX_EXTERNAL_ARTIFACTS ?= "${TOPDIR}/flex-external-artifacts"
 # Support for SOURCERY_VERSION_REQUIRED
 require conf/distro/include/sourcery-version-check.inc
 
-# Handle GPLv3 gdbserver
```

Comment:
I think we need ore information in commit summary for these changes

Reply:
This is already covered in the commit `local.conf.sample: remove binutils and gplv3 hacks`. This is just removing the bits the local.conf was referencing.

Reply:
I also added a mention of this in the commit.